src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs

```diff
@@ -198,6 +198,7 @@ private static bool NeedsEscaping(UnicodeCategory category)
                 case UnicodeCategory.ParagraphSeparator:
                 case UnicodeCategory.LineSeparator:
                 case UnicodeCategory.Surrogate:
+                case UnicodeCategory.Format:
```

Comment:
Can this method be shared between both languages? The implementation appears to be exactly the same